keps/sig-architecture/4330-compatibility-versions/kep.yaml

```diff
@@ -0,0 +1,49 @@
+title: Compatibility Versions
+kep-number: 4330
+authors:
+  - "@alexzielenski"
+  - "@jpbetz"
+  - "@liggitt"
+  - "@logicalhan"
+  - "@siyuanfoundation"
+owning-sig: sig-architecture
+participating-sigs:
+  - sig-api-machinery
+  - wg-lts
+status: provisional
```

Comment:
Make sure the KEP status is marked as `implementable` for latest-milestone: 1.30.